android/jni/com/mapswithme/maps/Framework.cpp

```diff
@@ -660,6 +660,11 @@ void Framework::SetupWidget(gui::EWidget widget, float x, float y, dp::Anchor an
   m_guiPositions[widget] = gui::Position(m2::PointF(x, y), anchor);
 }
 
+void Framework::UpdateMyPositionRoutingOffset(int offsetY)
+{
+  NativeFramework()->UpdateMyPositionRoutingOffset(false, offsetY);
```

Comment:
Why is this line can't be called directly from MapFragment.cpp? Why is another wrapper layer needed?